Packs/CofenseTriage/Integrations/CofenseTriage/CofenseTriage.py

```diff
@@ -224,28 +301,39 @@ def search_reports_command() -> None:
 
 def search_reports(subject=None, url=None, file_hash=None, reported_at=None, created_at=None, reporter=None,
                    verbose=False, max_matches=30) -> list:
+    # TODO move to new TriageReportQuery (or similar) class
     params = {'start_date': datetime.strftime(reported_at, TIME_FORMAT)}
-    reports = http_request(url_suffix='/processed_reports', params=params)
-
-    if not isinstance(reports, list):
-        reports = [reports]
-
-    reporters = []  # type: list
-    if reporter:
-        reporters = get_all_reporters(time_frame=min(reported_at, created_at))
+    reports = TRIAGE_INSTANCE.request("processed_reports", params=params)
 
     matches = []
 
     for report in reports:
-        if subject and subject != report.get('subject'):
+        if subject and subject != report.get('report_subject'):
+            # TODO do we really want to do exact string match here? not case-insensitive substring?
```

Comment:
case insensitive sounds better.

Reply:
I wanted to avoid making any functional changes to legacy behavior for this release. The only user-visible changes should be the two new commands.

I would love to address this in a future release, though.